recipes/tsl-hat-trie/all/conanfile.py

```diff
@@ -14,14 +14,14 @@ class TslHatTrieConan(ConanFile):
     topics = ("string", "trie", "structure", "hash map", "hash set")
     homepage = "https://github.com/Tessil/hat-trie"
     url = "https://github.com/conan-io/conan-center-index"
-
+    package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
 
     def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("tsl-array-hash/0.7.1")
+        self.requires("tsl-array-hash/0.7.1", transitive_headers=True)
```

Comment:
No need for transitive_headers in a `package_type = "header-library"` recipe, it's automatically inferred ([Docs here](https://docs.conan.io/2/reference/conanfile/methods/requirements.html#package-type-trait-inferring))

Approving either way as it does not change the behaviour of the program and we avoid recompiling it :)

Reply:
ok thanks, I was not sure.